java/android/AmpliApp/app/src/main/java/com/example/ampliapp/App.java

```diff
@@ -0,0 +1,21 @@
+package com.example.ampliapp;
+
+import android.app.Application;
+
+import com.amplitude.ampli.Ampli;
+import com.amplitude.ampli.LoadClientOptions;
+import com.amplitude.ampli.LoadOptions;
+
+public class App extends Application {
+    @Override
+    public void onCreate() {
+        super.onCreate();
+
+        String amplitudeApiKey = null; // Set your Amplitude API key
```

Comment:
Please external (see comment in Kotlin)

Reply:
Fixed

Reply:
Can you please move this to a dedicated `secrets.xml` like in Kotlin